### PR TITLE
Added documentation on Event Schema and Namespace guide

### PR DESCRIPTION
**Pull request by victorskl**
* Improved repo landing README.md

### CODE REVIEW

File: docs/event-schemas/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  "source": ["orcabus.bclconvertmanager"],
+  "detail": {
+    "status": ["SUCCEEDED"]
+  }
+}
+```
+
+## Namespace
+
+Service namespaces are for filtering Event Rule purpose. This is used in event `source` property when service emits messages. It denotes where the message is originating from. The convention follows reverse domain name. We follow "compat" format of [CDK stack directory name](../../lib/workload/stateless/stacks). i.e. Removing dash character from kebab-case.
```
Comment (reisingerf) [inline docs/event-schemas/README.md:87]: Ditto, also use detail-type (see naming of examples)

**Review by reisingerf — APPROVED** [review]
LGTM
(minor comments)

**Review by brainstorm — APPROVED** [review]
👍🏻

**Review by mmalenic — APPROVED** [review]

**Comment by victorskl** [comment]
We are going to rearrange this REAME a little bit in next PR. Merging it as-is for now and, doing minor edit in next PR.

Merging...

**Pull request merged**